app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -3,7 +3,13 @@ module VerifyProfileConcern
 
   def url_for_pending_profile_reason
     return idv_verify_by_mail_enter_code_url if current_user.gpo_verification_pending_profile?
-    return idv_in_person_ready_to_verify_url if current_user.in_person_pending_profile?
+    if current_user.in_person_pending_profile? 
```

Comment:
Was wondering if we want to do this for all instances except fraud, also going to ensure that when its in a pending format and u go through the account page, it still has no weirdness for users.